tests/test_message.py

```diff
@@ -542,6 +542,44 @@ def test_text_markdown_emoji(self):
         )
         assert expected == message.text_markdown
 
+    def test_text_html_custom_emoji(self):
```

Comment:
I think you can merge this test by parametrizing this test and the one below (test_text_markdown_custom_emoji) into one.